lib/Controller/ConfigController.php

```diff
@@ -303,24 +328,37 @@ private function setIntegrationConfig(array $values): array {
 				$this->clearUserInfo($userUID);
 			});
 		}
+
+
+		// whenever doing full reset we want the group folder switch state to be "on" in the UI
+		// so setting `fresh_group_folder_setup` as true
+		if ($runningFullReset) {
+			$this->config->setAppValue(Application::APP_ID, 'fresh_group_folder_setup', "1");
```

Comment:
Should we also change this to `project`?
```suggestion0
			$this->config->setAppValue(Application::APP_ID, 'fresh_project_folder_setup', "1");
```